emails/pos/receipt.html

```diff
@@ -0,0 +1,188 @@
+													<td style="border-bottom: 1px solid #D9D9D9; padding: 4px 0 4px 9px; text-align: right;">{TransactionReference}</td>
+													<td style="border-bottom: 1px solid #D9D9D9; padding: 4px 0 4px 9px; text-align: right;">{TransactionStatus}</td>
+													<td style="border-bottom: 1px solid #D9D9D9; padding: 4px 0 4px 9px; text-align: right;">{TransactionTotal}</td>
+												</tr>
+											}}',
+											after: '
+													</tbody>
+												</table>
+											'
+										)}
+									</div>
+									<div id="VatInfo" style="font-size: 12px; margin: 18px auto; text-align: center;">
+										{TaxSummary(
+											before: '
+												<table id="OrderTotalsByTaxRate" cellpadding="0" cellspacing="0" style="font-size: 12px; text-align: left; margin: 0 auto;">
+													<tr>
+														<th scope="col" style="border-bottom: 1px solid #444444; padding: 4px 0; text-align: left;">{%TaxRate}</th>
+														<th scope="col" style="border-bottom: 1px solid #444444; padding: 4px 0 4px 9px; text-align: right;">{%TotalWithoutTax}</th>
+														<th scope="col" style="border-bottom: 1px solid #444444; padding: 4px 0 4px 9px; text-align: right;">{%TotalTax}</th>
+														<th scope="col" style="border-bottom: 1px solid #444444; padding: 4px 0 4px 9px; text-align: right;">{%TotalWithTax}</th>
+													</tr>
+											',
+											after: '
+													<tr>
+														<th scope="row" style="border-bottom: 1px solid #D9D9D9; padding: 4px 0; text-align: left;">{%Total}</th>
+														<th style="border-bottom: 1px solid #D9D9D9; padding: 4px 0 4px 9px; text-align: right;">{OrderTotalWithoutTax}</th>
+														<th style="border-bottom: 1px solid #D9D9D9; padding: 4px 0 4px 9px; text-align: right;">{OrderTax}</th>
+														<th style="border-bottom: 1px solid #D9D9D9; padding: 4px 0 4px 9px; text-align: right;">{OrderTotal}</th>
+													</tr>
+												</table>
+											',
+											helper: {{
+												<tr>
+													<td style="border-bottom: 1px solid #D9D9D9; padding: 4px 0; text-align: left; font-weight: bold;">{TaxSummaryRowTaxRate(after:'%')}</td>
+													<td style="border-bottom: 1px solid #D9D9D9; padding: 4px 0 4px 9px; text-align: right;">{TaxSummaryRowWithoutTax}</td>
+													<td style="border-bottom: 1px solid #D9D9D9; padding: 4px 0 4px 9px; text-align: right;">{TaxSummaryRowTax}</td>
+													<td style="border-bottom: 1px solid #D9D9D9; padding: 4px 0 4px 9px; text-align: right;">{TaxSummaryRowTotal}</td>
+												</tr>
+											}}
+										)}
+										{OrderVATDescription}
+									</div>
+								</td>
+							</tr>
+						</table>
+					</td>
+					<td style="background: #f0f0f0 url({ShopUrl}{ThemeUrl}/i/email-sidebar-top.gif) repeat-x left top; border-left: 1px solid #e2e2e2;" valign="top" width="241">
+						<table cellpadding="18" cellspacing="0" border="0" bgcolor="#f0f0f0" height="100%" width="241">
```

Comment:
```suggestion
						<table cellpadding="18" cellspacing="0" border="0" bgcolor="#f0f0f0" width="241">
```

Vielä ei toiminut, mutta jos tuon tuosta poistaa niin näyttäisi pelittävän.